scripts/codegen-check-consistency.js

```diff
@@ -0,0 +1,8 @@
+const { checkCodegenIntegrity } = require('./codegen-utils');
+
+if (require.main === module) {
+  async function main() {
+    await checkCodegenIntegrity();
+  }
+  main();
```

Comment:
Cannot it be called directly?

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/2224/commits/1b2a000a3c5238fd142f95ddb7119e0a216f0ed8